server/controllers/users/searchUsers.js

```diff
@@ -0,0 +1,19 @@
+const { users: { searchUsers } } = require('../../database/queries');
+
+exports.searchUsers = (req, res, next) => {
+  // we will use req.query to get results based on it
+  const { search: searchQuery } = req.query;
+  searchUsers(searchQuery)
+    .then(({ rows }) => {
+      if (rows.length && rows) {
```

Comment:
pls turn the condition around to `if (rows && rows.length)` (if rows.length is true that means rows is automatically true as well and no need to check it, however if rows is null or undefined, you will get an error saying `property length does not exist for undefined` or sth like that

Reply:
oh yes good note